src/model/openthaigpt_pretraining_model/optimizers/lion/utils.py

```diff
@@ -206,7 +206,7 @@ def __init__(
             )
         else:
             raise NotImplementedError("only support lion or AdamW")
-        self.model = torch.compile(model)
+        self.model = torch.compile(model)  # type: ignore
```

Comment:
If I remembered correctly, you once reported that torch.compile is not working with Flash Attention/Lion.
Could you elaborate why it works now?

Reply:
It works correctly. I just told you that i cannot install torch 2 in kaggle meaning that i cannot use torch.compile and flash attention. But on Colab i can install it which cannot use ddp mode to test.